src/apps/chifra/pkg/output/stream.go

```diff
@@ -57,11 +55,15 @@ func StreamWithTemplate(w io.Writer, model types.Model, tmpl *template.Template)
 // StreamModel streams a single `Model`
 func StreamModel(w io.Writer, model types.Model, options OutputOptions) error {
 	if options.Format == "json" {
```

Comment:
Why, in some cases, do we test only for `Format == "json"` and in other we test both `Format == "json" || opts.IsRaw`? I would prefer if we forced `Format = "json"` if we parsed `--raw` from the command line. Is that possible?

Reply:
We need to check both conditions if the code is running before `InitGlobals` (which sets format to `"json"` always when `--raw` is present)